File: app/src/main/java/com/fthdgn/books/api/response/Volumes.java
package com.fthdgn.books.api.response;

import android.support.annotation.Keep;

import java.util.List;

@Keep
@SuppressWarnings({"unused", "WeakerAccess"})
public class Volumes {

    public static final String KIND_BOOKS_VOLUMES = "books#volumes";

    /**
     * {@link #KIND_BOOKS_VOLUMES}
     */
    public String kind;
    public int totalItems;
    public List<Volume> items;

}

File: app/src/main/java/com/fthdgn/books/api/response/Volumes.kt
@file:Suppress("unused")

package com.fthdgn.books.api.response

import android.support.annotation.Keep

@Keep
class Volumes {

    /**
     * [.KIND_BOOKS_VOLUMES]
     */
    var kind: String? = null
    var totalItems: Int = 0
    var items: List<Volume>? = null

    companion object {

        val KIND_BOOKS_VOLUMES = "books#volumes"
    }

}
